test/AutoRest.TestServer.Tests/Mgmt/CodeModelValidatorTest.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.IO;
+using AutoRest.CSharp.Input;
+using AutoRest.CSharp.Mgmt.AutoRest;
+using AutoRest.CSharp.Utilities;
+using NUnit.Framework;
+
+namespace AutoRest.TestServer.Tests.Mgmt
+{
+    public class CodeModelValidatorTest
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void EmptyOperationGroupName()
+        {
+            var model = loadModel("Mgmt/EmptyOperationGroupName.yaml");
```

Comment:
Can we create an instance of CodeModel here ourselves?

Reply:
For this case, yes. I was thinking in the future when we add more complex test cases, we need to load from a file v.s. object literal initializer.